packages/foundations/package.json

```
@@ -25,6 +26,7 @@
     "copy-build:readme": "cpr README.md ../../build-outputs/foundations/README.md -o",
     "copy-build:web": "cpr build ../../build-outputs/foundations/build -o -f \"(compose|ios-swift)\"",
     "copy:scss": "cpr scss build/scss -o",
+    "generate:fonts": "node scripts/generate-icon-fonts/index.js --src ./assets/icons/functional --variants solid inverted --prefix db_ic_ --fontName db-ux --withSizes true --debug true",
```

```suggestion
    "generate:icon-fonts": "node scripts/generate-icon-fonts/index.js --src ./assets/icons/functional --variants solid inverted --prefix db_ic_ --fontName db-ux --withSizes true --debug true",
```

What about renaming this for "clarity" ?